mkdocs.yml

```
@@ -71,8 +71,8 @@ extra_css:
   - stylesheets/extra.css
 
 nav:
+  - "Intro to DUB": index.md  # Moved to top level
```

```suggestion
  - "Intro to DUB": index.md
```

putting in a comment will just confuse people reading the code in the future when they expect it to be top level and haven't seen it anywhere else